src/theme/icons.tsx

```diff
@@ -498,6 +503,11 @@ const icons = {
       <path d="M5 4h14c.5523 0 1 .44772 1 1v14c0 .5523-.4477 1-1 1H5c-.55228 0-1-.4477-1-1V5c0-.55228.44772-1 1-1Z" />
     ),
   },
+  pending: {
```

Comment:
i would name this hourglass but pending works as well 
```suggestion
  hourglass: {
```